### PR TITLE
feat: Remove dashboard image

### PR DESCRIPTION
**Pull request by kylecarbs**
**Should not be merged before 1.19 is cut.**

Removes the dashboard image. Requires a change to the ingress... directing all HTTP traffic to cemanager.

### CODE REVIEW

**Review by f0ssel — APPROVED** [review]

File: templates/ingress.yaml [hunk 1]
```diff
@@ -372,9 +372,6 @@ metadata:
   {{- end }}
 spec:
 {{- include "coder.ingress.tls" . }}
-  backend:
-    serviceName: dashboard
```
Comment (deansheather) [inline templates/ingress.yaml:376]: Shouldn't this become cemanager 8080

**Review by deansheather — APPROVED** [review]
1.19 has already been cut

File: templates/ingress.yaml (hunk 1 above)
Comment (kylecarbs) [reply templates/ingress.yaml:376]: This appears to be an unused property... but @coadler is on the blame so he'll let us know

Comment (coadler) [reply templates/ingress.yaml:376]: This is the backend it sends traffic to when it matches none of the rules. It should probably be changed to cemanager

Comment (kylecarbs) [reply templates/ingress.yaml:376]: Gotcha. Will update.

**Pull request merged**